hive/src/main/java/io/confluent/connect/storage/hive/HiveSchemaConverter.java

```diff
@@ -27,11 +30,22 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.apache.kafka.connect.data.Time;
+import org.apache.kafka.connect.data.Timestamp;
+import org.apache.kafka.connect.errors.ConnectException;
 
 public class HiveSchemaConverter {
 
   private static final Map<Type, TypeInfo> TYPE_TO_TYPEINFO;
 
+  // the name has to be consistent with io.confluent.connect.avro.AvroData, when Connect Decimal
+  // schema is created, this property name is used to set precision.
+  // We have to use the exact name to retrieve precision value.
+  protected static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
+
+  // this is the maximum digits Hive allows for DECIMAL type.
+  protected static final int DECIMAL_PRECISION_DEFAULT = 38;
```

Comment:
Let's call this `HIVE_DECIMAL_PRECISION_MAX`
to distinguish it from `AvroData#CONNECT_AVRO_DECIMAL_PRECISION_DEFAULT` or similar
and also note that this is not a default of a configurable option but rather the max of a configurable option